documentation/README.md

```diff
@@ -32,10 +45,7 @@ $ git clone https://github.com/masak/007.git
 [...]
 ```
 
-### Setting an environment variable
-
-We're one step away from running our first 007 program. Before that, we need to
-set an environment variable `PERL6LIB`:
+Finally, we need to set an environment variable `PERL6LIB`:
```

Comment:
'environment variable'
Suggestion: 'path environment variable'

Reply:
I think there is too strong an association with the environment variable actually named `PATH` for the term "path environment variable" to be a net improvement in clarity.

Is there prior art somewhere? I can't recall hearing someone write "path environment variable" before. If there's plenty of examples of other texts using that term for a certain kind of environment variable, I might reconsider.